sale_product_set_variant/wizard/product_set_add.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2017-2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError, ValidationError
+from odoo.addons import decimal_precision as dp
+
+
+class ProductSetAddLine(models.TransientModel):
+    _name = 'product.set.add.line'
+    _order = 'sequence'
+
+    wiz_id = fields.Many2one(
```

Comment:
@bizzappdev IMO add field string

Reply:
Wiz id is not added into any view and also the default string will work. so IMO not sure it should be added or not

Reply:
> Wiz id is not added into any view and also the default string will work. so IMO not sure it should be added or not

Yes, it's also not blocking issues but If you add it's better for a reviewer

Reply:
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#38fields

Reply:
Yes, I know that. :smile: 

But It's String look like  Ugly `Wiz Id`

Reply:
Yes, it will look ugly only if it is added in the view. IMO string is not needed since the field is not added in the view.

Reply:
But where will it be displayed?

Reply:
Leave it. It is not needed.

Reply:
Skip it as it is not needed since the field is not added in the view.